epub33/core/index.html

```diff
@@ -4362,123 +4320,118 @@ <h4>Scripting Contexts</h4>
+							<code>src</code> attribute makes no difference to its executing context.</p>
 
-						<p>and the following file <code>scripted02.xhtml</code>:</p>
+					<p>Which context Authors use for their scripts affects both what actions the scripts can perform and
+						the likelihood of support in Reading Systems, as described in the following subsections.</p>
 
-						<pre>&lt;html …&gt;
-    &lt;head&gt;
-        …
-        &lt;script type="text/javascript" href="slideshow.js"&gt;&lt;/script&gt;
-    &lt;/head&gt;
-    &lt;body&gt;
-        …
-    &lt;/body&gt;
-&lt;/html&gt;</pre>
+					<div class="note">
+						<p>Refer to <a href="#scripted-contexts-example"></a> for an example of the difference between
+							the two contexts.</p>
+					</div>
+
+					<section id="sec-scripted-container-constrained">
+						<h5>Container-Constrained Scripts</h5>
 
-						<p>From these examples, it is true that:</p>
+						<p>A <dfn>container-constrained script</dfn> is either of the following:</p>
 
 						<ul>
 							<li>
-								<p>the code in the <code>script</code> element in the <code>head</code> in
-										<code>scripted01.xhtml</code> is a spine-level script because the document is
-									referenced from the spine; and</p>
+								<p>An instance of the [[!HTML]] <a
+										href="https://html.spec.whatwg.org/multipage/scripting.html#the-script-element"
+											><code>script</code></a> element contained in an <a>XHTML Content
+										Document</a> that is embedded in a parent XHTML Content Document using the
+									[[!HTML]] <a
+										href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element"
+											><code>iframe</code></a> element.</p>
 							</li>
 							<li>
-								<p>the code in the <code>script</code> element in <code>scripted02.xhtml</code> is a
-									container-constrained script because the HTML file it occurs in is included in
-										<code>scripted01.xhtml</code> via the <code>iframe</code> element.</p>
+								<p>An instance of the [[!SVG]] <a
+										href="https://www.w3.org/TR/SVG/interact.html#ScriptElement"
+										><code>script</code></a> element contained in an <a>SVG Content Document</a>
+									that is embedded in a parent XHTML Content Document using the [[!HTML]] <a
+										href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element"
+											><code>iframe</code></a> element.</p>
 							</li>
 						</ul>
-					</aside>
-				</section>
 
-				<section id="sec-scripted-container-constrained">
-					<h4>Container-Constrained Scripts</h4>
+						<p id="confreq-cd-scripted-container">A container-constrained script MUST NOT contain
+							instructions for modifying the DOM of the EPUB Content Document that embeds it (i.e., the
+							one that contains the <code>iframe</code> element). It also MUST NOT contain instructions
+							for manipulating the size of its containing rectangle.</p>
 
```

Comment:
The problem with these statements is that I do not see how this can be tested. I presume epubcheck would be unable to do that, it would require to understand the behavior of the script.

I am completely out of my comfort zone here, maybe what I say is very different. Looking at the [iframe `sandbox` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) isn't it possible to replace this (or make it more precise) by applying constraints on the attributes the `<iframe>` element should or should not use? _That_ can be tested by epubcheck.

I am just wondering; maybe @rdeltour, @danielweck, or @bduga can comment on this...

Reply:
Can you open a separate issue for this as it would change existing normative requirements?

I don't want to open a new discussion in a PR.

Reply:
You are right. I will.

pinging @dauwhe and @wareid because we just had some discussions about this...

Reply:
done in #1523